book/release-service.md

```diff
@@ -31,13 +31,13 @@ The Release service is dependent on the following services:
 
 The diagram below shows the interaction of the release service and other services.
 
-![](../diagrams/hacbs-data-flow.jpg)
+![](../diagrams/release-service/hacbs-data-flow.jpg)
 
 ## Application Context
 
 The diagram below shows the flow of custom resources and the orchestration of pipelines by the release service.
 
-![](../diagrams/hacbs-release-service-dataflow.jpg)
+![](../diagrams/release-service/hacbs-release-service-dataflow.jpg)
```

Comment:
There is inconsistency in the naming -- this is `dataflow` while the other is `data-flow`. Also both are `HACBS`.

Reply:
renamed to be `data-flow`. I'd prefer to leave this one in the release-service folder for now